terraform/070-id-sync/vars.tf

```diff
@@ -113,3 +113,7 @@ variable "alb_dns_name" {
 variable "notifier_email_to" {
   type = "string"
 }
+
+variable "sync_safety_cutoff" {
+  default = ""
```

Comment:
Having an empty default value could cause us trouble with the cron file's env vars, just as a heads-up. It may be better to make it match the default in ID Sync (0.15, if I remember right).

Reply:
ok sure, that sounds good